index.html

```diff
@@ -40,7 +40,7 @@ <h2 id="main-h2">Private Digital Currency</h2>
                     <div class="row middle-xs info-block-row private">
                         <div class="col-lg-6 col-md-6 col-sm-8 col-xs-12  why-text">
                             <h3>Monero is secure.</h3>
-                            <p>Monero can't be hacked to steal your funds, due to the power of distributed consensus, which you can read about here. This means that you are responsible for your money, and don't have to trust any entity to keep it safe for you.</p>
+                            <p>Monero cannot be hacked to steal your funds, due to its decentralized nature. Only you are responsible for securing your Monero.</p>
```

Comment:
> the second sentence doesn't need to change

I think the whole line probably should be rewritten. Maybe:
`Due to it's decentralized nature, Monero cannot be hacked or stolen. The power of distributed consensus means that transactions are immutably recorded on the blockchain and you don't have to trust anyone to keep your Monero safe.`

(Distributed consensus link is gone on the live site... probably should just link to Moneropedia entry of the same name if it exists or it can be written.)